SwedbankPaySDK/Classes/SwedbankPaySDK+Extensions/CallbackHandling.swift

```diff
@@ -16,10 +16,21 @@
 import Foundation
 
 public extension SwedbankPaySDK {
+    /// Call from your `UIApplicationDelegate(_:open:options:)`
+    /// implementation to forward paymentUrls to the SDK
+    ///
+    /// - parameter url: the URL to forward
+    /// - returns: `true` if the url was successfully processed by the SDK, `false` otherwise (e.g. if the url was not an active payment url)
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 141 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)